resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md

```diff
@@ -131,7 +131,7 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% endtab %}
 {% endtabs %}
 
-In-depth instructions can be seen at the [Analytics Readme](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/b7706ec53f082a5385a6eb5870418af7c116f0b1/using-mobile-extensions/adobe-analytics/readme.md#add-analytics-to-your-app).
+In-depth instructions can be seen at the [Analytics Readme](../../using-mobile-extensions/adobe-analytics/README.md#add-analytics-to-your-application).
```

Comment:
Let's reword this to:
`For more details, follow the steps for [adding Analytics to your application](../../using-mobile-extensions/adobe-analytics/README.md#add-analytics-to-your-application).`

Reply:
Updated as 
```
For more details, see [Add Analytics to your application](...).
```